bears/vcs/git/GitCommitBear.py

```diff
@@ -66,6 +98,7 @@ def run(self, allow_empty_commit_message: bool = False, **kwargs):
             self.err('git:', repr(stderr))
             return
 
+        body = stdout.rstrip('\n')
```

Comment:
actually this is not the commit body, it also contains the shortlog. Also you are doing double work with the stmt below:
```
stdout =s tdout.ristrip('\n').splitlines()
```
For consistency I would suggest you also introduce a `shortlog` variable:
```
shortlog = ...
body = stdout[:stdout.find('\n')] # <-- this is not idiot proof, you need to check empty commit messages,
# single line commit message and multiline commit message for proper results.
...
yield from self.check_shortlog(shortlog, ...)
yield from self.check_body(body.splitlines())
yield from self.check_issue(body)
```